pyLRT/RadTran.py

```diff
@@ -112,7 +106,108 @@ def run(self, verbose=False, print_input=False, print_output=False, regrid=True,
             return (np.genfromtxt(io.StringIO(process.stdout)),
                     _read_verbose(io.StringIO(process.stderr), regrid=regrid))
         return np.genfromtxt(io.StringIO(process.stdout))
-
+    
+    def add_cloud(
+        self, type="liquid", height=None, base_height=None, thickness=None, 
```

Comment:
Would it be possible to add some kind of flag/option that states the asusmptions being made when determining the cloud profile (e.g. 'adiabatic', 'vertically homogeneous'). We wouldn't have to fill in the other options now, but that could be useful.

Reply:
Nice idea, I'll have a look at what assumptions LRT makes about the profile internally